java/code/webapp/WEB-INF/nav/system_detail.xml

```diff
@@ -14,7 +14,7 @@
     <rhn-tab name="Proxy" url="/rhn/systems/details/ProxyClients.do" acl="system_is_proxy()">
     </rhn-tab>
     <rhn-tab name="Reactivation" url="/rhn/systems/details/Activation.do" acl="system_feature(ftr_agent_smith)"/>
-    <rhn-tab name="Hardware">
+    <rhn-tab name="Hardware" acl="not system_has_foreign_entitlement() or not system_is_proxy()">
```

Comment:
Is not enough to filter only by the new entitlement?

Reply:
the new entitlement is also used for Virtual Host Managers... I don't want to break anything with it